### PR TITLE
Export cleanup?

### PR DESCRIPTION
**Pull request by emord**
@esoergel I was doing some code investigation for something very unrelated, but I found all of these seemingly unused methods. my gut feeling is that they had some relation to an old exports framework, but I don't have anything concrete that links it to that. 

Some of this stuff is from 2012, so its not even clear if its from the export framework you recently did some cleanup on or not, but htought you may be a bit more sure if these are still used.

### CODE REVIEW

**Review by esoergel — APPROVED** [review]
Your assumptions look right - this seems like parts of various old exports.  There's still a ton of that left, I've been extricating pieces as I find time and am feeling annoyed, but there's a bunch of code intertwined with current exports

**Comment by emord** [comment]
@esoergel you're on interrupt this week and know the most about exports, so I'll let you merge this when you're ready.

**Pull request merged**
